.github/ISSUE_TEMPLATE/release.md

```diff
@@ -1,59 +1,59 @@
 ---
-name: k6 Release 
-about: k6 release accommodates activities and a checklist that has the k6 open-source release process.
+name: k6 Release
+about: k6 release accommodates activities and a checklist with the k6 open-source release process.
 title: 'k6 release v0.4x.0'
```

Comment:
```suggestion
title: 'k6 release v0.4x.0'
labels: ["release"]
```
I don't know if it is really useful, but we are already using it and I like the color of the label. So, why not? :stuck_out_tongue: